src/AlgAssAbsOrd/Ideal.jl

```diff
@@ -1503,7 +1503,11 @@ function Base.one(S::AlgAssAbsOrdIdlSet)
   return ideal(order(S), one(order(S)))
 end
 
- ==(x::AlgAssAbsOrdIdlSet, y::AlgAssAbsOrdIdlSet) = x.order === y.order
+==(x::AlgAssAbsOrdIdlSet, y::AlgAssAbsOrdIdlSet) = x.order === y.order
+
+function Base.hash(x::AlgAssAbsOrdIdlSet, h::UInt)
+  return xor(hash(order(x), h), 0x3d4177ad98877e36 % UInt)
```

Comment:
```suggestion
  return hash(objectid(x.order), h)
```
since the `==` delegates to `===` on `.order`, we can make the hash stronger here by using `objectid` (which is basically the hash wrt `===`)

Reply:
Isn't this the default hash anyway?

Reply:
yes, the `hash` fallback delegated to `objectid`. But here, if at some point there is a specific hash function for the type of `x.order`, this would be used. This would both be slower than just `objectid` and it would produce a weaker hash